multi-int-store/src/main/java/com/connexta/multiintstore/common/ServerException.java

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) 2019 Connexta, LLC
+ *
+ * Released under the GNU Lesser General Public License version 3; see
+ * https://www.gnu.org/licenses/lgpl-3.0.html
+ */
+package com.connexta.multiintstore.common;
+
+public class ServerException extends RuntimeException {
```

Comment:
✏️ Improve the name of ServerException and ClientException to be more specific to this app, like `MetadataRetrievalServerException`